creator-node/src/services/stateMachineManager/index.js

```diff
@@ -72,6 +91,89 @@ class StateMachineManager {
     this.highestEnabledReconfigMode = highestEnabledReconfigMode
     this.enabledReconfigModesSet = enabledReconfigModesSet
   }
+
+  /**
+   * After a monitorState job completes, this function gets called to
+   * enqueue findPotentialSyncs and findReplicaSetUpdates jobs, which each
+   * find state anomalies that need to be reconciled for the slice of users that
+   * the monitorState job queried.
+   * @param {Object} monitorStateJobResult the monitorState job that successfully completed
+   */
+  _enqueueFindPotentialSyncsAndReplicaSetUpdates(monitorStateJobResult) {
+    const {
+      users,
+      unhealthyPeers,
+      replicaSetNodesToUserClockStatusesMap,
+      userSecondarySyncMetricsMap
+    } = monitorStateJobResult
+
+    QueueInterfacer.addStateMonitoringJob(
+      JOB_NAMES.FIND_REPLICA_SET_UPDATES,
+      /** data */
+      {
+        users,
+        unhealthyPeers,
+        replicaSetNodesToUserClockStatusesMap,
+        userSecondarySyncMetricsMap
+      }
+    )
+    QueueInterfacer.addStateMonitoringJob(
+      JOB_NAMES.FIND_POTENTIAL_SYNCS,
+      /** data */
+      {
+        users,
+        unhealthyPeers,
+        replicaSetNodesToUserClockStatusesMap,
+        userSecondarySyncMetricsMap
+      }
+    )
+  }
+
+  /**
+   * After a findPotentialSyncs job completes, this function gets called to
+   * enqueue an enqueueSyncRequests job, which takes each potential sync request
+   * and adds it to the queue to be issued if it's needed (based on the clock values)
+   * @param {Object} findPotentialSyncsJobResult the findPotentialSyncs job that successfully completed
+   */
+  _enqueueEnqueueSyncRequests(findPotentialSyncsJobResult) {
```

Comment:
nitname

Reply:
changed -- this function is no longer needed at all with the new way of jobs returning other jobs that get enqueued